runtime/README.md

```diff
@@ -1030,29 +1443,201 @@ The DPE `GET_CERTIFICATE_CHAIN` command shall return the following certificates:
+|                                | Public Key   | DPE Alias Public Key value                                          |
+| Signature Algorithm Identifier | Algorithm    | ecdsa-with-SHA384                                                   |
+|                                | Parameters   | Named Curve = prime384v1                                            |
+| Signature Value                |              | Digital signature for the certificate                               |
+| KeyUsage                       | keyCertSign  | 1                                                                   |
+| Basic Constraints              | CA           | False                                                               |
+| Policy OIDs                    |              | id-tcg-kp-attestLoc                                                 |
+| tcg-dice-MultiTcbInfo\*        | FWIDs        | [0] "Journey" TCI Value                                             |
+|                                |              | [1] "Current" TCI Value. Latest `INPUT_DATA` made by DeriveContext. |
+|                                | Type         | 4-byte TYPE field of TCI node                                       |
+|                                | VendorInfo   | Locality of the caller (analog for PAUSER)                          |
 
 \*MultiTcbInfo contains one TcbInfo for each TCI Node in the path from the
 current TCI Node to the root. Max of 32.
+
+
+## Manifest-Based Image Authorization (1.2)
+
+Caliptra's goal is to enable integrators and vendors to meet standard security requirements for creating cryptographic identity and securely reporting measurements through DICE and DPE Certificate chains and Caliptra-owned private-public key pairs. In addition, Caliptra 1.0 provides an `ECDSA384_SIGNATURE_VERIFY` command to enable an SoC RoT to verify its own FW signatures so that it can develop an SoC secure boot using Caliptra cryptography. Caliptra 1.1 expanded the verify command to a PQC-safe `LMS_SIGNATURE_VERIFY` command. In each of these cases, it is left up to the vendor to ensure that they build a secure environment for introducing and verifying FW integrity and authenticity and then executing mutable FW.
+
+The Caliptra Measurement manifest feature expands on Caliptra-provided secure verifier abilities. The Measurement Manifest feature provides a standard Caliptra-supported definition to enable the following use cases for integrators, vendors, and owners.
+
+* Caliptra-Endorsed Aggregated Secure Boot
+* Caliptra-Endorsed Local Verifier
+
+Each of these abilities are tied to Caliptra Vendor and Owner FW signing keys and should be independent of any SoC RoT FW signing keys.
+
+Manifest-based image authorization is implemented via three mailbox commands: [`SET_AUTH_MANIFEST`](#set-auth-manifest), [`SET_IMAGE_METADATA`](#set-image-metadata), and [`AUTHORIZE_AND_STASH`](#authorize-and-stash).
+
+### Caliptra-Endorsed Aggregated Secure Boot
```

Comment:
wondering if this should be called "Aggregated Measured Boot"

Reply:
The confusion here in calling it "Secure Boot" is that when Caliptra is Passive, it has no "power" to enforce what it is running